### ISSUE TITLE
UI element to add new resource

### ISSUE THREAD

**Issue by dschulten** [smollweide/node-mock-server#38]
The UI can only be used to add new responses once the following folder structure exists:
```
resource-url-segment
  #path
    +--- method
      +--- mock 
```
It would be very useful to have an Add Resource button in the UI which asks for the resource url, #path and method so that I do not have to create the above structure manually. Similar to:

```
Resource: ______
Path: #_____________ (separate path segments with #)
Method: ___ (GET, POST ...)
[Create]
```

**Comment by smollweide** [comment]
Hi @dschulten,
good point, I will take care of it soon.
Thanks

**Comment by smollweide** [comment]
Implemented with release [0.16.0](https://github.com/smollweide/node-mock-server/releases/tag/0.16.0)
